github/filecoin-project.yml

```diff
@@ -5048,13 +5048,17 @@ teams:
         - Kubuxu
         - ZenGround0
   fvm-crate-owners:
+    description: Defines ownership of various crates per
```

Comment:
@galargh : I see this isn't applied via terraform.  Is that a github-as-code deficiency or terraform?  Should I create a backlog item?